pkgs/development/python-modules/qscintilla-qt5/default.nix

```
@@ -15,6 +15,10 @@ buildPythonPackage {
   buildInputs = [ qscintilla ];
   propagatedBuildInputs = [ pyqt5 ];
 
+  postPatch = ''
+    sed -e "s#target_config.py_module_dir#'$out/${python.sitePackages}'#" -i Python/configure.py
```

```suggestion
    substituteInPlace Python/configure.py \
      --replace "target_config.py_module_dir" "$out/${python.sitePackages}"
```
Then it'll throw a warning when the source changes and it doesn't replace anything anymore.

---

Updated. Thanks.